openedx/core/djangoapps/programs/utils.py

```diff
@@ -3,13 +3,16 @@
 import datetime
 import logging
 from urlparse import urljoin
+from pytz import utc
```

Comment:
Nitpick: pytz is a third-party package and belongs in the second block of imports, where it was before. It's not part of the standard library.